tools/generate-helpers/pg-table-bindings/migration.go.tpl

```diff
@@ -49,6 +50,19 @@ var (
 				return errors.Wrap(err,
 					"moving {{.Table|lowerCase}} from rocksdb to postgres")
 			}
+			{{- if gt (len .Schema.RelationshipsToDefineAsForeignKeys) 0 }}
+			if err := prune(databases.PostgresDB); err != nil {
+				return errors.Wrap(err,
+				"pruning {{.Table|lowerCase}}")
+			}
+			// Now that migrations are complete, turn the constraints back on
```

Comment:
We do not need to turn on the constraints in each migration. We can turn them on altogether after persist.